src/funcCoreTools/uninstallFuncCoreTools.ts

```diff
@@ -3,28 +3,36 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { IAzureQuickPickItem } from 'vscode-azureextensionui';
 import { funcPackageName, PackageManager } from '../constants';
 import { ext } from '../extensionVariables';
 import { localize } from '../localize';
 import { cpUtils } from '../utils/cpUtils';
-import { getFuncPackageManager } from './getFuncPackageManager';
-import { funcToolsInstalled } from './validateFuncCoreToolsInstalled';
+import { getFuncPackageManagers } from './getFuncPackageManagers';
 
-export async function uninstallFuncCoreTools(): Promise<void> {
+export async function uninstallFuncCoreTools(packageManagers?: PackageManager[]): Promise<void> {
     ext.outputChannel.show();
-    if (await funcToolsInstalled()) {
-        switch (await getFuncPackageManager(true /* isFuncInstalled */)) {
-            case PackageManager.npm:
-                await cpUtils.executeCommand(ext.outputChannel, undefined, 'npm', 'uninstall', '-g', funcPackageName);
-                break;
-            case PackageManager.brew:
-                await cpUtils.executeCommand(ext.outputChannel, undefined, 'brew', 'uninstall', funcPackageName);
-                break;
-            default:
-                throw new Error(localize('cannotUninstall', 'Uninstall is only supported for brew or npm.'));
-                break;
-        }
+    // tslint:disable-next-line: strict-boolean-expressions
+    packageManagers = packageManagers || await getFuncPackageManagers(true /* isFuncInstalled */);
+    let packageManager: PackageManager;
+    if (packageManagers.length === 0) {
+        throw new Error(localize('notInstalled', 'Cannot uninstall Azure Functions Core Tools because it is not installed with brew or npm.'));
+    } else if (packageManagers.length === 1) {
+        packageManager = packageManagers[0];
     } else {
-        throw new Error(localize('notInstalled', 'Cannot uninstall Azure Functions Core Tools because it is not installed.'));
+        const placeHolder: string = localize('multipleInstalls', 'Multiple installs of the func cli detected. Select the one to uninstall');
+        const picks: IAzureQuickPickItem<PackageManager>[] = packageManagers.map(pm => { return { label: pm, data: pm }; });
+        packageManager = (await ext.ui.showQuickPick(picks, { placeHolder })).data;
+    }
+
```

Comment:
Since the wording of the prompt asks for which install to remove, I think there should should be a confirmation. 

A couple of questions: 
1.  The package manager enum shows only npm and brew. Are there only two possibilities? 
2. If so, the options ["Keep npm" and "Keep brew" ] or ["Uninstall npm" and "Uninstall brew"] would be less prone to human error (for all the users who don't fully read the prompt message :P)

Reply:
In this case, the command that was invoked was UninstallFuncCoreTools which is kind of the confirmation in itself.  We can assume that the user fully intends to uninstall by calling this command since it is only exposed via the command pallette. (Sorry for responding, just happened to approve this PR as soon as you commented 😆)

Reply:
1. There's a few more, but those are the only two we support
1. To add to what Nathan said - it's pretty easy to re-install the func cli, so I'm not that worried about a confirmation message.

I'm okay with making the picks more explicit, but I don't like just "Uninstall npm" because that makes it seem like I'm uninstalling _npm_ instead of _func_. Changed to "Uninstall npm package"

![Screen Shot 2019-03-20 at 2 57 11 PM](https://user-images.githubusercontent.com/11282622/54721950-a1f90900-4b20-11e9-8078-c186981fad50.png)